docs/mast/mast.rst

```diff
@@ -927,6 +938,22 @@ and returns a target pixel file, with format described
      1  PIXELS        1 BinTableHDU    280   1196R x 12C   [D, E, J, 25J, 25E, 25E, 25E, 25E, J, E, E, 38A]
      2  APERTURE      1 ImageHDU        81   (5, 5)   int32
 
+For users with time-sensitive targets who would like cutouts from the latest observations, 
+we recommend requesting for the TICA product. Using the same target from the example above, 
+let's request for the TICA cutouts:
+
+.. doctest-remote-data::
+
+   >>> from astroquery.mast import Tesscut
+   >>> from astropy.coordinates import SkyCoord
+   ...
+   >>> cutout_coord = SkyCoord(107.18696, -70.50919, unit="deg")
+   >>> hdulist = Tesscut.get_cutouts(coordinates=cutout_coord, size=5, product='tica')
```

Comment:
I'd suggest noting in the narrative docs above that `size=5` means 5 pixels, and perhaps even mention how big a pixel (typically?) is

Reply:
If this is specified above (I haven't read the full docs tonight), perhaps just remove `size=5`, since it is the default value.

Reply:
The unit is specified a couple examples above, so I've removed size=5 and added that the default is 5 pixels in the little blurb where it's specified.